src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java

```diff
@@ -52,7 +53,9 @@
 public class PluginProcessor extends BaseComponentProcessor<PluginDescriptor> implements
         EntandoK8SServiceReportableProcessor {
 
-    public static final String PLUGIN_DEPLOYMENT_PREFIX = "pn-";
+    private static final String PRIMARY_TENANT_CODE = "primary";
```

Comment:
Please use the constant `EntandoMultiTenancy.PRIMARY_TENANT` from the new version of entando-k8s-custom-model